concent_api/core/validation.py

```diff
@@ -104,7 +105,13 @@ def validate_task_to_compute(task_to_compute: message.TaskToCompute) -> None:
     validate_hex_public_key(task_to_compute.provider_public_key, 'provider_public_key')
     validate_hex_public_key(task_to_compute.requestor_public_key, 'requestor_public_key')
     validate_secure_hash_algorithm(task_to_compute.package_hash)
-    validate_positive_integer_value(task_to_compute.price)
+    try:
```

Comment:
please extract this `try...except` block to a separate function, and then unit-test that new function instead

Reply:
Done